package.json

```diff
@@ -68,7 +68,7 @@
     "wicg-inert": "^3.0.1"
   },
   "devDependencies": {
-    "@11ty/eleventy": "^0.11.0",
+    "@11ty/eleventy": "^0.11.1",
```

Comment:
Need access to `config.on('afterBuild', () => ...)`

Reply:
So it looks like this upgrade is causing issues with `pngquant`
```
[11:49:07] Using gulpfile ~/src/web.dev/gulpfile.js
[11:49:07] Starting 'build'...
[11:49:07] Starting 'copy-global-images'...
[11:49:07] Starting 'copy-misc'...
[11:49:07] Starting 'copy-content-assets'...
[11:49:07] Starting 'copy-fonts'...
[11:49:07] Starting 'sass'...
[11:49:08] Finished 'sass' after 1.1 s
[11:49:08] 'copy-global-images' errored after 1.45 s
[11:49:08] Error in plugin "gulp-imagemin"
Message:
    
Details:
    errno: -2
    code: ENOENT
    syscall: spawn /Users/asraine/src/web.dev/node_modules/pngquant-bin/vendor/pngquant
    path: /Users/asraine/src/web.dev/node_modules/pngquant-bin/vendor/pngquant
    spawnargs: -,--quality,100-100
    stdout: 
    stderr: 
    failed: true
    signal: null
    cmd: /Users/asraine/src/web.dev/node_modules/pngquant-bin/vendor/pngquant - --quality 100-100
    timedOut: false
    killed: false
    fileName: /Users/asraine/src/web.dev/src/images/android-chrome-192x192.png
    domainEmitter: [object Object]
    domainThrown: false

[11:49:08] 'build' errored after 1.46 s
[11:49:08] The following tasks did not complete: copy-misc, copy-content-assets, copy-fonts
[11:49:08] Did you forget to signal async completion?
```
I was using `npm i` to upgrade the package. Is there another way I should be doing it?

Reply:
I think this was an issue, with how I was bumping the version. https://github.com/GoogleChrome/web.dev/commit/83fd829250eb364d4d73e67ed5ffc571e547722d appears to have fixed this.